index.html

```diff
@@ -16,7 +16,7 @@
 
 <body id="index-body">
```

Comment:
### USE GOOD HTML CODE FLOW.
- It's not necessary to add an id to the body since there is only one body.
-  Please also Articulate all your HTML tags properly.